http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -2914,6 +2912,16 @@ components:
         cloudService:
           type: string
           enum: [gce, dataproc]
+    OneOfRuntimeConfig
```

Comment:
Leo swagger isn't rendering this was merged, I think the cause is a missing `:` here… fixing.

Reply:
PR: https://github.com/DataBiosphere/leonardo/pull/1609